src/app/modules/common/services.js

```diff
@@ -521,6 +521,7 @@ angular.module('BloomLibraryApp.services', ['restangular'])
             this.modifyBookField = function(book, field, value) {
                 var rBook = restangular.withConfig(authService.config()).one('classes/books', book.objectId);
                 rBook[field] = value;
+                rBook['updateSource'] = 'web';
```

Comment:
This is a change that can currently only be made by an authorized user, right, since the only changes that can be made through the BL app (apart from deleting books) are from the grid view.
However, eventually there will probably be changes ordinary users can make, such as adding reviews and ratings and  private or shared tags.
At that point I think it might be useful to distinguish a book that was last updated by the librarian, or a moderator, or whatever it is someone currently has to be to make these changes.
Anticipating that, how about instead of making updateSource here simply 'web' we actually indicate whatever it is we know about who did it...'moderator' or 'admin' or 'librarian' or whatever is appropriate?